### PR TITLE
Juliet/fix/protocol

### PR DESCRIPTION
**Pull request by junyezhu**
## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [x] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by cormacpayne** [comment]
@junyezhu what is being changed in this PR? Please add a snippet to the Network change log with an overview of what you're changing

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

**Comment by cormacpayne** [comment]
@azuresdkci add to whitelist

**Comment by junyezhu** [comment]
@cormacpayne added. We found out although we already added protocolConfiguration, we didn't pass any protocol value into backend.

File: src/ResourceManager/Resources/Commands.Resources/ChangeLog.md [hunk 1]
```diff
@@ -26,6 +26,7 @@
 * Support managed identity on policy assignments.
 * Parameters with default values are no longer requred when assigning a policy with `New-AzureRmPolicyAssignment`
 * Add new cmdlet Get-AzureRmPolicyAlias for retrieving policy aliases
+* Update cmdlet Test-AzureRmNetworkWatcherConnectivity, pass the protocol value to backend.
```
Comment (cormacpayne) [inline src/ResourceManager/Resources/Commands.Resources/ChangeLog.md]: @junyezhu this needs to go in the [Network change log](https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Network/Commands.Network/ChangeLog.md) under the `## Current Release` header

**Pull request merged**